openstack_controller/manifest.json

```
@@ -14,7 +14,7 @@
     "mac_os",
     "windows"
   ],
-  "public_title": "Datadog-Openstack_controller Integration",
+  "public_title": "Datadog-Openstack Controller Integration",
```

```suggestion
  "public_title": "Openstack Controller",
```
We can also drop the `Datadog- Integration` parts since we don't use that in integration templates either.